typescript-conditionals/main.ts

```diff
@@ -0,0 +1,140 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+function isUnderFive(x: number): boolean {
+  if (x < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+const isUnderFiveResult: boolean = isUnderFive(3);
+console.log('value of isUnderFiveResult:', isUnderFiveResult);
+
+function isEvenNumber(x: number): boolean {
+  if (x % 2) {
+    return true;
+  } else {
+    return false;
+  }
+}
+const isEvenNumberResult: boolean = isEvenNumber(10);
+console.log('value of isEvenNumber:', isEvenNumberResult);
+
+const string1: string = 'Jennifer';
+
+function startsWithJ(string: string): boolean {
+  if (string.charAt(0) === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+const startsWithJResult: boolean = startsWithJ(string1);
+console.log('value of startsWithJResult:', startsWithJResult);
+
+const person = {
+  name: 'Bryant',
+  age: 22,
+};
+
+function isOldEnoughToDrink(person: { name: string; age: number }): boolean {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+const isOldEnoughToDrinkResult: boolean = isOldEnoughToDrink(person);
+console.log('value of is oldEnoughToDrink:', isOldEnoughToDrinkResult);
+
+function isOldEnoughToDrive(person: { name: string; age: number }): boolean {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+const isOldEnoughToDriveResult: boolean = isOldEnoughToDrive(person);
+console.log('value of isOldEnoughToDrive:', isOldEnoughToDriveResult);
+
+function isOldEnoughToDrinkAndDrive(person: {
+  name: string;
+  age: number;
+}): boolean {
+  if (person.age >= 21) {
+    return false;
+  } else {
+    return false;
+  }
+}
+
+const isOldEnoughToDrinkAndDriveResult: boolean =
+  isOldEnoughToDrinkAndDrive(person);
+console.log(
+  'value of isOldEnoughToDrinkAndDrive:',
+  isOldEnoughToDrinkAndDriveResult
+);
+
+function categorizeAcidity(x: number): string {
+  if (x === 7) {
+    return 'neutral';
+  } else if (x > 7 && x < 14) {
+    return 'acid';
+  } else if (x < 7 && x > 0) {
+    return 'base';
+  } else {
+    return 'invalid pH level';
+  }
+}
+
+const categorizeAcidityResult: string = categorizeAcidity(9);
+console.log('value of categorizeAcidityResult:', categorizeAcidityResult);
+
+function introduceWarnerBro(name: string): string {
+  switch (name) {
+    case 'yakko':
+      return 'hello';
+    case 'wakko':
+      return 'hello';
+    case 'dot':
+      return 'hello';
+    default:
+      return 'BYE';
+  }
+}
```

Comment:
Your function does not return the values from the prompt.

There is also a way to make multiple case statement evaluate into the same return. I would recommend trying to do this for this switch statement.